src/style.css

```diff
@@ -1,3 +1,90 @@
-body {
+@font-face {
+  font-family: Roboto;
+  src: url('/fonts/Roboto-Medium.ttf') format('truetype');
```

Comment:
Ensure that the font file path is correct and that the font file is available at the specified location. This is crucial for the font to load properly.